webpack/scenes/AlternateContentSources/Create/Steps/ACSReview.js

```diff
@@ -16,9 +17,14 @@ const ACSReview = () => {
   return (
     <>
       <WizardHeader
-        title={__('Review Details')}
-        description={__('Review the information below and click Add to add your source. ' +
-                'To edit details in previous steps, click Back or any step on the left.')}
+        title={__('Review details')}
+        description={
+          <>
+            {__('Review the information below and click ')}<b>{__('Add')}</b>{__(' to add your source. ' +
+                'To edit details in previous steps, click ')}<b>{__('Back')}</b>{__(' or any step on the left.')
```

Comment:
The `<b>` tags should be `<strong>` instead; see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b